pyproject.toml

```diff
@@ -59,6 +60,9 @@ click = "^8.1"
 Mako = "^1.2"
 ni-python-styleguide = "^0.4"
 sphinx_rtd_theme = "1.0"
+poetry-grpc-plugin = "^0.1.0"
+grpcio = { version = "^1.49.0", optional = true }
+protobuf = "^3.20"
```

Comment:
I had to downgrade this version because I was getting the error shown below and one of the suggestions was to downgrade this. I hope this is the right way to resolve this. Please share your comments on this @bkeryan.
![image](https://user-images.githubusercontent.com/97511689/231448366-7d5a051a-bdb0-4fb4-bdb1-77aa25b8a078.png)

Reply:
No, this is not the right way to resolve this. I'll look into it.